sdcm/cluster.py

```diff
@@ -2925,7 +2952,8 @@ def wait_for_init(self, node_list=None, verbose=False, timeout=None):  # pylint:
         :param timeout: timeout in minutes to wait for init to be finished
         :return:
         """
-        node_list = node_list or self.nodes  # pylint: disable=no-member
+        # if not node_list, first init seeds (self.seed_nodes), and after non seeds (self.non_seed_nodes)
+        node_list = node_list or self.seed_nodes + self.non_seed_nodes
```

Comment:
just for protocol, `node_setup` is run in  `wait_for_init_wrap` so the reorder of the node setup will not work here. 
move the logic to setup and create separate method that starts seeds first and the other nodes